tests/test_load_azd_env.py

```diff
@@ -29,53 +30,55 @@ def _azd_env_set(key: str, value: str, *, cwd: Path) -> str:
 
 
 def test_load_azd_env(tmp_path: Path) -> None:
-    from os import getenv
 
     with open(tmp_path / "azure.yaml", "w") as config:
         config.write("name: dotenv-azd-test\n")
 
     _azd_env_new("MY_AZD_ENV", cwd=tmp_path)
     var_set = load_azd_env(cwd=tmp_path)
-    assert getenv("AZURE_ENV_NAME") == "MY_AZD_ENV"
+    assert os.getenv("AZURE_ENV_NAME") == "MY_AZD_ENV"
     assert var_set
 
 
-def test_load_azd_env_override(tmp_path: Path) -> None:
-    from os import environ, getenv
+def test_load_azd_env_override(tmp_path: Path, monkeypatch) -> None:
 
     with open(tmp_path / "azure.yaml", "w") as config:
         config.write("name: dotenv-azd-test\n")
 
-    environ["VAR1"] = "INITIAL"
+    monkeypatch.setenv("VAR1", "INITIAL")
     _azd_env_new("MY_AZD_ENV", cwd=tmp_path)
     _azd_env_set("VAR1", "OVERRIDE", cwd=tmp_path)
     var_set = load_azd_env(cwd=tmp_path)
-    assert getenv("AZURE_ENV_NAME") == "MY_AZD_ENV"
-    assert getenv("VAR1") == "INITIAL"
+    assert os.getenv("AZURE_ENV_NAME") == "MY_AZD_ENV"
```

Comment:
Just changed to os.getenv to make it clear its coming from os

Reply:
Isn't it better to narrow down what's imported? but maybe it's not as important as for Typescript, I'm not so intimate with the subtelties in Python of importing a module versus an exported object of the module

Reply:
I don't think we get the same benefits as in TypeScript, though its a good question. I like the readability of seeing where a function came from, and for os, I feel like we always "import os". I'll ask around.
Functionality wise, nothing should be affected.

Reply:
Ok